packages/frontend/core/src/desktop/dialogs/doc-info/info-modal.tsx

```diff
@@ -72,6 +72,16 @@ export const InfoTable = ({
     });
   }, []);
 
+  const onPropertyChange = useCallback(
+    (property: DocCustomPropertyInfo, _value: unknown) => {
+      track.$.docInfoPanel.property.editProperty({
+        type: property.type,
+        control: 'at menu',
```

Comment:
Why `at menu` here? Any relationship with doc info?

Reply:
indeed. the control here should be removed.